Resources/Private/Build/Behat/composer.json

```diff
@@ -1,8 +1,7 @@
 {
 	"require": {
 		"behat/behat": "^3.5",
-		"behat/mink-extension": "^2.3",
-		"behat/mink-goutte-driver": "*",
```

Comment:
if you do this, then you will end up running *all* tests with selenium, which is so damn slow that it hurts. See my comment in https://github.com/neos/behat/issues/33 with a solution to replace "goutte" with "browserkit" for a pure PHP browser implementation. This way test suites which do not require `@javascript` run very fast.

Reply:
and also: don't forget to also adapt the `behat.yml.dist` to remove goutte (and maybe add browserkit)

Reply:
Good point, I probably mistook the **dev** dependency of `friends-of-behat/mink-extension` on `behat/mink-goutte-driver` for a regular one and wanted to remove a transitional dependency.